datafusion/functions/src/unicode/translate.rs

```diff
@@ -120,7 +144,7 @@ fn translate<T: OffsetSizeTrait>(args: &[ArrayRef]) -> Result<ArrayRef> {
             }
             _ => None,
         })
-        .collect::<GenericStringArray<T>>();
+        .collect::<StringArray>();
```

Comment:
Will this cause an issue at all? To be honest I'm not entirely sure if there will be a perf issue by swapping out GenericStringArray and StringArray.

Reply:
Actually I will likely need to revert to GenericStringArray<T> so I can use i64 as a type on it by the looks StringArray is i32.